### ISSUE TITLE
Ads load but not seen

### ISSUE THREAD

**Issue by whale** [reallygoodemails/really-good-emails-frontend#57]
The ads load in a way that when I scroll they aren't seen because they load wehere the page has already scrolled past. Any ideas on ways to fix that?

**Comment by danielmcormond** [comment]
Yes, right now the way we position ads is a little hacky. Remember how we used to mix them into the grid, but then we couldn't control their position? Now instead we add them to the page next to the grid, using absolute positioning to anchor them where we want them. I think right now the horizontal spacing is hard-coded to a specific pixel value, which is why you're seeing that behavior.

There are a couple things we can do: 1) dynamically calculate the horizontal spacing based on the user's browser window size 2) load more ads earlier so they are already positioned before the user scrolls past them. I have an older MacBook Air (non-retina) which is probably why I haven't seen this yet. I can test at home on my retina MacBook Pro and fine tune it.

**Comment by whale** [comment]
Pre loading could work, or in an ideal world having the ads be sticky on 
the side as you scroll.

**Comment by danielmcormond** [comment]
@whale try it now and let me know what you think

**Comment by whale** [comment]
@danielmcormond seems much much better. How difficult would it be to have this stick to the top? Overall much better though.

**Comment by whale** [comment]
@danielmcormond can you chime in on making the ads sticky on scroll and swamp when the new one comes in? Just don't know if that's a big or a small change.
